packages/google_identity_services_web/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.1.0
+
+* Initial open-source release.
```

Comment:
How about just "Initial release."? This makes it sound like this is pre-existing closed-source code.

Reply:
Good call!